src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int RANDOM_NUMBERS = 100;
```

Comment:
```suggestion
    public static final int MAX_FIGURE_COUNT = 100;
```
let's have more informative names for constants

Reply:
done